modules/secured-cluster-configuration-options-operator.adoc

```diff
@@ -56,6 +56,15 @@ The default value is `false`.
 | `admissionControl.resources.requests`
 | Use this parameter to override the default resource requests for the admission controller.
 
+| `admissionControl.bypass`
+| Use this parameter to bypass the admission controller.
+
+| `admissionControl.contactImageScanners`
+| Specify `true` to enable inline scanning of images not already scanned during a deployment's admission review.
```

Comment:
nit: maybe "...of images that are not already..."  Up to you, clearly not wrong.